torch/_dynamo/variables/functions.py

```diff
@@ -193,6 +193,11 @@ def call_function(
                 tx, self.fn, self.get_name(), options, args, kwargs
             )
 
+        if self.fn == is_tracing:
```

Comment:
You should be able to get this behavior with:
https://github.com/pytorch/pytorch/blob/f28927e9c4694819f95c5dda2b111b0ff6dbd606/torch/_dynamo/config.py#L54

Reply:
Thanks, this looks way simpler